docs/advanced-topics.asciidoc

```diff
@@ -1,10 +1,37 @@
 [[advanced-topics]]
 == Advanced Topics
 
+* <<longtasks>>
+* <<typescript>>
 * <<custom-transactions, Custom Transactions>>
 * <<custom-transaction-name, Initial page load transaction names>>
-* <<typescript>>
 
+[[longtasks]]
+=== How to interpret long task spans in the UI
+
+Long tasks is a new performance metric that can be used for measuring the
+responsiveness of an application and helps developers to understand the bad user
+experience. It enables detecting tasks that monopolize the UI thread for
+extended periods (greater than 50 milliseconds) and block other
+critical tasks from being executed as stated in the https://github.com/w3c/longtasks[official spec].
```

Comment:
What if we swapped the what and the why? Something like:
```
Long tasks enables the detection of tasks that monopolize the
UI thread for extended periods of time (greater than 50 milliseconds) and block other
critical tasks from being executed -- as stated in the https://github.com/w3c/longtasks[official spec].
This new performance metric is useful for measuring the responsiveness of an application,
and can help developers better understand bad user experiences.
```

Reply:
Don't really oppose your suggestion here. But shouldn't we start with explaining what is long task and then go ahead with why its intereting?.